examples/random/src/main.cpp

```diff
@@ -20,9 +20,7 @@
 #include <iostream>
 #include <vector>
 
-std::uint32_t seed = 777;
-
-template <typename RealType>
+template <::std::uint32_t Seed, typename RealType>
 void scalar_example(sycl::queue& queue, std::vector<RealType>& x) {
```

Comment:
From my point of view the previous solution was better. There are no sense to put seed as a template parameter

Reply:
Fixed